Changes about electronic give out for SIMCards.

lukaszkarykowski opened the pull request:
Added modules from BackOffice transitions to SIMCards module.
This modules allows to accept give out of SIMCards electronically.

lukaszkarykowski committed:
Changes about electronic give out for SIMCards.

Added modules from BackOffice transitions to SIMCards module.
This modules allows to accept give out of SIMCards electronically.

ar4s commented on src/ralph/admin/templatetags/dashboard_tags.py:
```diff
@@ -54,6 +56,18 @@ def get_user_equipment_to_accept_tile_data(user):
     }
 
 
+def get_user_simcard_to_accept_tile_data(user):
```
Could you provide some test for this function? Hint: mock ``get_simcards_to_accept``to simplify testing.

lukaszkarykowski replied:
Test passed, and pushed to git.

ar4s commented on src/ralph/sim_cards/models.py:
```diff
@@ -157,6 +165,101 @@ def assign_user(cls, instances, **kwargs):
+        pass
+
+    @classmethod
+    def _generate_report(cls, name, requester, instances, language):
```
DRY - extract `_generate_report` to common function and use it in `back_office/model.py` and here.

lukaszkarykowski replied:
Changed. And I push with changes.

lukaszkarykowski committed:
Changes about electronic give out for SIMCards.

Added modules from BackOffice transitions to SIMCards module.
This modules allows to accept give out of SIMCards electronically.

lukaszkarykowski committed:
Changes about electronic give out for SIMCards.

Added modules from BackOffice transitions to SIMCards module.
This modules allows to accept give out of SIMCards electronically.

lukaszkarykowski committed:
Changes about electronic give out for SIMCards.

Added modules from BackOffice transitions to SIMCards module.
This modules allows to accept give out of SIMCards electronically.

lukaszkarykowski committed:
Changes about electronic give out for SIMCards.

Added modules from BackOffice transitions to SIMCards module.
This modules allows to accept give out of SIMCards electronically.
